site/docusaurus.config.js

```diff
@@ -66,7 +66,7 @@ const theButton = require('./src/rehype/thebutton');
           // other options are passed to posthog-js init as is
         },
       ],
-      ['docusaurus-plugin-iasql', {}],
+      ['docusaurus-plugin-iasql', { usedSidebar: 'docs' }],
```

Comment:
what's the result of this change?

Reply:
the problem was that when you were inside a doc, it didn't have any sidebar. Now it will display just the current sidebar as all the documents.

Reply:
oh, I see, I've reproduced it